examples/computer_vision/detr_coco_pytorch/const_fake.yaml

```diff
@@ -63,5 +63,6 @@ searcher:
         batches: 100 
 resources:
     shm_size: 2000000000
+    slots_per_trial: 2
```

Comment:
sorry, you resolved this comment but did not respond or make any changes to the pr?

Reply:
sorry about that

Reply:
If my memory isn't failing me (... and it is reasonably likely that it is) you have resolved this comment twice without either answering my question or changing the PR?

Please stop resolving comments when you respond; let the person who has no further response be the one who resolves the comment.  That makes it easier to avoid situations like this.

Reply:
sorry,  somehow got back in again

Reply:
> sorry, didn't know that it's the reviewer that should resolve issues

No need to apologize, I don't think there's any clear standard.  But I definitely have a strong preference, because otherwise re-reviewing means clicking "show-resolved" on every single resolved comment to see if I agree with the resolution.  Turns long reviews into like an O(N^2) operation.  And also, I don't think it has to be the reviewer, I think it's the person who reads the last response and doesn't want to respond who should resolve them.